include/cbmqtm/cbmqtm_dbdistributor.php

```diff
@@ -32,30 +32,34 @@ private static function setDB() {
 		static::$db = new PearDatabase();
 	}
 
-	public function sendMessage($channel, $producer, $consumer, $type, $share, $sequence, $expires, $deliverafter, $userid, $information) {
+	public function sendMessage($channel, $producer, $consumer, $type, $share, $sequence, $expires, $deliverafter, $userid, $information, $deliverRange=array('deliverStartTime'=>'', 'deliverEndTime'=>'', 'canSendOnSaturday'=>'', 'canSendOnSunday'=>'')) {
```

Comment:
this initialization is incorrect because if the programmer sends you a partial array:

`['canSendOnSaturday'=>0]`

which is valid, the other values are not initialized and will cause warnings and errors below. Please initialize the values individually.

Reply:
`
public function sendMessage($channel, $producer, $consumer, $type, $share, $sequence, $expires, $deliverafter, $userid, $information, $deliverRange = array()) {
`